FrontDesk/src/FrontDesk.Api/Endpoints/File/Read.cs

```diff
@@ -24,9 +24,9 @@ public Read()
         ]
         public override async Task<ActionResult<FileItem>> HandleAsync([FromRoute]string fileName, CancellationToken cancellationToken)
         {
-            if (string.IsNullOrEmpty(fileName)) return BadRequest();            
+            if (string.IsNullOrEmpty(fileName)) return BadRequest();
 
-            var fullPath = Path.Combine(Directory.GetCurrentDirectory(), "images", "Patients", fileName);
+            var fullPath = Path.Combine(Directory.GetCurrentDirectory(), "Images", "Patients", fileName);
```

Comment:
Let's try and use lower-case folder and file names if possible so we don't need to worry about case sensitivity across windows/linux.

Reply:
done